### DISCUSSION TITLE
i can't connect to my localhost

### DISCUSSION THREAD

**Discussion by donghuiyong** [Difegue/LANraragi#440] [Q&A]
![image](https://user-images.githubusercontent.com/62844440/113397742-2220fd00-93d0-11eb-8100-bdf02abc0f32.png)
What's the problem?

**Comment by Difegue** [comment]
¯\\\_(ツ)_/¯ How the heck am I supposed to know ?  
I'm afraid this is a bit too vague, needs more details such as which OS, which install type, what's in the logs, etc etc.  

If you're using the Windows installer, this discussion might be of interest: https://github.com/Difegue/LANraragi/discussions/378

**Reply by donghuiyong** [reply]
Thank you. I changed the port and it worked. Now I have another question. I have a proxy client running on windows which provides a socks5 proxy(localhost:7890). Can I make LRR go through this proxy?
